frontend/src/pages/NewFile/components/NewProjectCard/newProjectCardStyle.ts

```diff
@@ -1,12 +1,13 @@
 import { styled } from 'goober'
 
-export const CardContainer = styled('button')`
+export const CardContainer = styled('button')<{height: number}>`
   display: grid;
   grid-template-rows: 1.3fr 1fr;
   box-sizing: border-box;
   min-width: 18em;
   width: 20em;
-
+  height: ${p => p.height || '100%'}px;
```

Comment:
In case there is a problem in the state, we just set the height to 100% so its least usable